### PR TITLE
chore: adds a quivr case study

### PR DESCRIPTION
**Pull request by kiwicopple**
- https://zone-www-dot-com-git-chore-customerquivr-supabase.vercel.app/customers
- https://zone-www-dot-com-git-chore-customerquivr-supabase.vercel.app/customers/quivr

### CODE REVIEW

File: apps/www/_customers/quivr.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+In May of 2023, [Stan Girard's](https://twitter.com/_StanGirard) started building small prototypes that allowed him to “chat with documents”. After 2 weeks of research, he settled on an idea - build a “second brain” where a user could dump all their digital knowledge (audio, URLs, text, and code) into a vector store and query it with GPT4.
+
+He built the first version in a single afternoon, pushed it to GitHub, and then [tweeted about it](https://twitter.com/_StanGirard/status/1657021618571313155?s=20). One viral tweet later, and [Quivr](https://github.com/StanGirard/quivr) was born.
+
+## Choosing a vector database
+
+A critical piece of the tech stack was the vector store. Stan needed a place to store millions of embeddings. After comparing between Supabase, Pinecone, and Chroma, he settled on [Supabase Vector](https://supabase.com/vector), our open source vector offering for developing AI applications. The decision was driven largely by his familiarity with Postgres, and the tight integration with Vercel.
+
+<Quote img="caryn-marooney.jpeg" caption="Stan Girard, Founder of Quivr.">
```
Comment (kiwicopple) [inline apps/www/_customers/quivr.mdx:42]: This needs a new avatar

File: apps/www/_customers/quivr.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    storing over 1.6 million embeddings and the performance and results are great. Open source
+    develop can easily contribute thanks to the SQL syntax known by millions of developers.
+  </p>
+</Quote>
+
+## Building an open source community
+
+It didn't take long for the Quivr community to grow. After the viral launch, the [Quivr repo](https://github.com/StanGirard/quivr) stayed at number 1 on [GitHub Trending](https://github.com/trending) for 4 days. Today, it has over 22,000 GitHub stars and 67 contributors. Supabase has been a key part of the open source stack since the beginning.
+
+<Quote img="caryn-marooney.jpeg" caption="Stan Girard, Founder of Quivr.">
```
Comment (kiwicopple) [inline apps/www/_customers/quivr.mdx:54]: This needs a new avatar

File: apps/www/_customers/quivr.mdx (hunk 1 above)
Comment (kiwicopple) [reply apps/www/_customers/quivr.mdx:42]: @StanGirard let us know which image to use here

Comment (StanGirard) [reply apps/www/_customers/quivr.mdx:42]: You can use the one on my lk. Not the best but I'll open a pr when I get a better one 😊 https://www.linkedin.com/in/stanislas-girard

File: apps/www/_customers/chatbase.mdx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  [
+    { label: 'Use case', text: 'AI chatbot builder' },
+    { label: 'Solutions', text: 'Supabase Database, Supabase Auth, Supabase Storage, Supabase Realtime' },
+  ]
+about: "Chatbase is an AI chatbot builder. It trains ChatGPT on your data and lets you add a chat widget to your website. Just upload a document or add a link to your website and get a chatbot that can answer any question about their content."
+---
+
+After getting a new grad offer from his “dream company” rescinded with tech layoffs, Yasser Elsaid decided to embark on a different path and bootstrap his own venture.
+
+Two months before the release of the ChatGPT API, Yasser explored the GPT 3 API and saw its immense potential. Leveraging Supabase as the backend, he built Chatbase in two weeks and launched it in February.
```
Comment (kiwicopple) [inline apps/www/_customers/chatbase.mdx:32]:
```suggestion
Two months before the release of the ChatGPT API, Yasser explored the GPT3 API and saw its immense potential. Leveraging Supabase as the backend, he built Chatbase in two weeks and launched it in February.
```

Do we also want to add links to Chatbase here?

File: apps/www/_customers/chatbase.mdx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Just five months after launching, Chatbase achieved $1,000,000 ARR in 5 months, making it one of the most successful single-founder AI products.
```
Comment (kiwicopple) [inline apps/www/_customers/chatbase.mdx:34]:
```suggestion
Just five months after launching, Chatbase reach $1,000,000 annualized revenue, making it one of the most successful single-founder AI products in the industry.
```

File: apps/www/_customers/chatbase.mdx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+## The Challenge
+
+Fascinated by the possibilities of GPT 3, Yasser envisioned Chatbase as an AI-driven chatbot capable of handling complex customer queries in real time. However, building a cost-effective and reliable solution to ingest and store data and manage large-scale customer interactions by himself wasn’t trivial.
```
Comment (kiwicopple) [inline apps/www/_customers/chatbase.mdx:38]:
```suggestion
Fascinated by the possibilities of GPT3, Yasser envisioned Chatbase as an AI-driven chatbot capable of handling complex customer queries in real time. However, building a cost-effective and reliable solution to ingest and store data and manage large-scale customer interactions by himself wasn’t trivial.
```

File: apps/www/_customers/chatbase.mdx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+  </p>
+</Quote>
+
+## What he built
+
+Chatbase is a powerful AI chatbot builder that enables users to train ChatGPT on their own data, revolutionizing the way businesses and individuals interact with customers. With seamless integration to over 5000 apps through Zapier, including popular platforms like Excel, Notion, Whatsapp, Discord, and Slack, Chatbase empowers everyone to deploy custom AI for their specific workflows.
+
+By simply uploading a PDF containing the desired information, Chatbase creates intelligent chatbots that automate customer support, handle frequently asked questions, and streamline communication channels, ultimately enhancing overall efficiency and user experience.
+
+Chatbase became a sensation and crossed 2,000,000 visitors in a matter of months and now has 2,200 paying customers.
```
Comment (kiwicopple) [inline apps/www/_customers/chatbase.mdx:62]:
```suggestion
Chatbase became a sensation, crossing 2,000,000 visitors in a matter of months and now has 2,200 paying customers.
```

File: apps/www/_customers/chatbase.mdx [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+## The Results
+
+Leveraging Supabase as the backend gave Yasser the speed and efficiency to turn into a significant advantage over the competition, enabling him to gain a strong advantage in the market and establish Chatbase as one of the pioneering AI chatbot builders.
+
+As Chatbase gained traction and acquired users, Supabase's robust infrastructure seamlessly accommodated the surging user base, ensuring smooth scalability and uninterrupted growth.
+
+Within just five months, Chatbase achieved an astonishing $1,000,000 in ARR.
```
Comment (kiwicopple) [inline apps/www/_customers/chatbase.mdx:70]:
```suggestion
Within just five months, Chatbase grew to an astonishing $1,000,000 in annualize revenue.
```

File: apps/www/_customers/chatbase.mdx [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+Chatbase is still bootstrapped, but now Yasser has a team of 5 people who help with customer support, marketing, and development. He is now exploring Supabase's capabilities for achieving GDPR compliance and implementing Vault for end-to-end encryption.
```
Comment (kiwicopple) [inline apps/www/_customers/chatbase.mdx:72]:
```suggestion
Chatbase is still bootstrapped, but now Yasser has a team of 5 who help with customer support, marketing, and development. He is now exploring Supabase's capabilities for achieving GDPR compliance and implementing Vault for end-to-end encryption.
```

Add links to Vault?

File: apps/www/_customers/chatbase.mdx [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+> If you want to learn more about how Supabase can help you build and scale AI apps with ease, reach out to us.
```
Comment (kiwicopple) [inline apps/www/_customers/chatbase.mdx:74]:
```suggestion
> If you want to learn more about how Supabase can help you build and scale AI apps with ease, reach out to us.
```
add links?

**Review by fsansalvadore — APPROVED** [review]

**Pull request merged**